packages/cli.cmd.typescript/package.json

```diff
@@ -23,6 +23,7 @@
     "clean": "rm -rf lib dist types build tsconfig.tsbuildinfo",
     "fix-lint": "eslint . --fix && dprint fmt --config $(find-up dprint.json)",
     "lint": "eslint . && dprint check  --config $(find-up dprint.json)",
+    "test": "vitest",
```

Comment:
this needs to be vitest run right? or else vitest defaults to watch mode?

Reply:
Good call!